pensjon-brevbaker/src/test/kotlin/no/nav/pensjon/brev/maler/example/LetterExample.kt

```diff
@@ -73,8 +73,8 @@ object LetterExample : StaticTemplate {
                         ifNotNull(tillegg1) {
                             item {
                                 textExpr(
-                                    Bokmal to "Du har fått tilleg1 for ".expr() + navn + " på ".expr() + it.str() + " Kr",
-                                    Nynorsk to "Du har fått tilleg1 for ".expr() + navn + " på ".expr() + it.str() + " Kr",
+                                    Bokmal to "Du har fått tilleg1 for ".expr() + navn + " på ".expr() + it.format() + " Kr",
+                                    Nynorsk to "Du har fått tilleg1 for ".expr() + navn + " på ".expr() + it.format() + " Kr",
```

Comment:
Kun eksempelbrevet, men burde vi ikke bruke kronefrase i den for at eksempel blir nærmere virkeligheten?

Reply:
Ikke så farlig. Jeg kunne tatt å lagt til litt flere features inn i brevet og retta på det i samme slengen.